### PR TITLE
Documentation: Update broken link

### PR DESCRIPTION
**Pull request by SavPhill**
I have removed the link titled 'How do I enable the remote API for dockerd' which points to: https://success.docker.com/article/how-do-i-enable-the-remote-api-for-dockerd as the success.docker.com domain is no longer available.

## What?

The documentation reference to success.docker.com is no longer available and therefore this causes a broken link in our own documentation.

## Why?

To prevent users accessing a broken link or incorrect link when trying to find addition documentation.

## How?

I have looked for an alternative article in docs.docker.com, but I did not find the article or a close match.

### CODE REVIEW

**Comment by alexstine** [comment]
Hmm, this seems tricky to me. I'm not sure we want to be in the business of just showing people how to modify their Docker system files without real links to docs or at least some trustworthy resource. Any of the docs watchers on this PR know if this might be okay for outside linking? I really don't know how that works.

Here are the guidelines set by the Docs Team but not sure if Gutenberg follows similar or different guidelines.
https://make.wordpress.org/docs/style-guide/linking/external-links/

I found this site to give a nice overview.
https://scriptcrunch.com/enable-docker-remote-api/

However, this is not for all operating systems. It seems this is purely a Linux bug.

Some part of me actually wonders if this bug still exists. Kind of odd to see nothing mentioned about it in official docs.

**Comment by SavPhill** [comment]
@alexstine Giving this PR a bump as it's 1 year without movement. What action should we take or should we discard this?

**Comment by alexstine** [comment]
CC: @noahtallen @ObliviousHarmony

File: docs/getting-started/devenv/docker-ubuntu.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ If docker is not running, try to start the service using:
 sudo systemctl start docker.service
 ```
 
-If docker is running, then it is not listening how the WordPress environment is trying to communicate. Try adding the following service override file to include listening on tcp. See docker documentation, [How do I enable the remote API for dockerd](https://success.docker.com/article/how-do-i-enable-the-remote-api-for-dockerd)
+If docker is running, then it is not listening how the WordPress environment is trying to communicate. Try adding the following service override file to include listening on tcp.
```
Comment (noahtallen) [inline docs/getting-started/devenv/docker-ubuntu.md]:
```suggestion
If docker is running, then it is not listening how the WordPress environment is trying to communicate. Try adding the following service override file to include listening on tcp. See [this Docker documentation](https://docs.docker.com/config/daemon/remote-access/) on how to configure remote access for Docker daemon.
```

**Review by noahtallen — CHANGES_REQUESTED** [review]
I think I found the right link: https://docs.docker.com/config/daemon/remote-access/

**Comment by SavPhill** [comment]
> I think I found the right link: https://docs.docker.com/config/daemon/remote-access/

Thanks Noah

**Review by noahtallen — APPROVED** [review]

**Pull request merged**